pom.xml

```diff
@@ -75,7 +75,7 @@ under the License.
         <maven-javadoc-plugin.version>3.3.2</maven-javadoc-plugin.version>
         <git-commit-id-maven-plugin.version>5.0.0</git-commit-id-maven-plugin.version>
 
-        <operator.sdk.version>5.1.1</operator.sdk.version>
+        <operator.sdk.version>5.1.2</operator.sdk.version>
```

Comment:
We need to update the notice files as well with the updated dependencies

Reply:
fixed, thx!